setup/index.js

```diff
@@ -80,12 +82,19 @@ function Setup() {
         return caps;
       }
 
+      function getProxy(){
+        if (proxyDetails) {
+          return proxy[proxyDetails.method].apply(proxy, proxyDetails.args);
```

Comment:
I would say let's add an inner condition here, to validate the user input somewhat:
```
if (proxyDetails.method && proxy[proxyDetails.method]) {
  return proxy[proxyDetails.method].apply(proxy, proxyDetails.args);
} else {
  //either this
  error('proxy configuration is incomplete or does not match the selenium-webdriver/proxy API');
  return proxy.direct();
  //or this
  throw new Error('nemo: proxy configuration is incomplete or does not match the selenium-webdriver/proxy API');
}
```